core/src/raw/adapters/kv/api.rs

```diff
@@ -18,17 +18,28 @@
 use std::fmt::Debug;
 use std::future::ready;
 
+use futures::stream::Empty;
 use futures::Future;
+use futures::Stream;
 
 use crate::raw::*;
 use crate::Capability;
 use crate::Scheme;
 use crate::*;
 
+/// A noop placeholder for Adapter::ScanIter
+pub type EmptyScanIter = Empty<Result<String>>;
```

Comment:
We can implement `kv::Scan` for `()` as empty.

Reply:
Done.